core/src/main/scala/org/http4s/ResponseCookie.scala

```diff
@@ -117,4 +120,175 @@ final case class ResponseCookie(
+
+    /* cookie-pair       = cookie-name "=" cookie-value */
+    val cookiePair = (cookieName <* char('=')) ~ cookieValue
+
+    /* sane-cookie-date  = <rfc1123-date, defined in [RFC2616], Section 3.3.1> */
+    val saneCookieDate = Rfc2616.rfc1123Date.flatMap { date: Rfc1123Date =>
+      import date._
+      try {
+        val dt = ZonedDateTime.of(year, month, day, hour, min, sec, 0, ZoneOffset.UTC)
+        pure(org.http4s.HttpDate.unsafeFromZonedDateTime(dt))
+      } catch {
+        case _: DateTimeException =>
+          failWith(s"Invalid sane-cookie-date: $year-$month-$day $hour:$min:$sec")
+      }
+    }
+
+    /* expires-av        = "Expires=" sane-cookie-date */
+    val expiresAv = ignoreCase1("Expires=").with1 *> saneCookieDate.map {
+      expires: HttpDate => (cookie: ResponseCookie) => cookie.withExpires(expires)
+    }
+
+    /* non-zero-digit    = %x31-39
+     *                   ; digits 1 through 9
+     */
+    val nonZeroDigit = charIn(0x31.toChar to 0x39.toChar)
+
+    /* max-age-av        = "Max-Age=" non-zero-digit *DIGIT
+     *                   ; In practice, both expires-av and max-age-av
+     *                   ; are limited to dates representable by the
+     *                   ; user agent.
+     */
+    val maxAgeAv = ignoreCase1("Max-Age=") *> (nonZeroDigit ~ rep(digit)).string.map {
+      maxAge: String => (cookie: ResponseCookie) => cookie.withMaxAge(maxAge.toLong)
+    }
+
+    /* domain-value      = <subdomain>
+     *                   ; defined in [RFC1034], Section 3.5, as
+     *                   ; *enhanced by [RFC1123], Section 2.1
+     */
+    def domainValue = Rfc1034.subdomain
+
+    /* domain-av         = "Domain=" domain-value */
+    val domainAv = ignoreCase1("Domain=") *> domainValue.string.map {
+      domain: String => (cookie: ResponseCookie) => cookie.withDomain(domain)
+    }
+
+    /* av-octet = %x20-3A / %x3C-7E
+     *          ; any CHAR except CTLs or ";"
+     *
+     * as proposed by https://www.rfc-editor.org/errata/eid3444
+     */
+    val avOctet = charIn(0x20.toChar to 0x3a.toChar)
+      .orElse1(charIn(0x3c.toChar to 0x7e.toChar))
+
+    /* path-value = *av-octet
+     *
+     * as amended by https://www.rfc-editor.org/errata/eid3444
+     */
+    val pathValue = avOctet.rep.string
+
+    /* path-av           = "Path=" path-value */
+    val pathAv = ignoreCase1("Domain=") *> pathValue.map { case path: String =>
+      (cookie: ResponseCookie) => cookie.withPath(path)
+    }
+
+    /* secure-av         = "Secure" */
+    val secureAv = ignoreCase1("Secure").as { (cookie: ResponseCookie) =>
+      cookie.withSecure(true)
+    }
+
+    /* httponly-av       = "HttpOnly" */
+    val httponlyAv = ignoreCase1("HttpOnly").as { (cookie: ResponseCookie) =>
+      cookie.withHttpOnly(true)
+    }
+
+    /* samesite-value    = "Strict" / "Lax" / "None"
+     * from https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-06
```

Comment:
This is from an expired draft spec with no successor.  @hamnis, does this still exist in the real world?  I think this was your addition.

Reply:
I see the draft spec is active in in [http-extensions](https://github.com/httpwg/http-extensions/issues?q=label%3A6265bis+).

Reply:
it is very much in the real world. Chrome requires same-site to be set for third-party cookies.